models/multiphase/d3q27_pf_velocity/Dynamics.R

```diff
@@ -6,17 +6,6 @@ if (Options$q27){
 	source("d3q27q15.R")
 }
 
-if (Options$ML){
```

Comment:
Was this intended  to be removed? Why? The same applies for 3D model

Reply:
decided that the ML was rather a specific niche thing currently used for testing and not ready for develop branch